tests/modules/encounters/resources/test_modify_encounter.py

```diff
@@ -62,9 +65,49 @@ def test_modify_encounter(db, flask_app_client, researcher_1, researcher_2, admi
     )
     assert res.json['id'] == str(new_encounter_1.guid)
 
+    # Attach some assets and annotations
+    from app.modules.asset_groups.models import AssetGroup
+
+    assets = AssetGroup.query.get(test_asset_group_uuid).assets
+    new_encounter_1.sighting.add_assets(assets)
```

Comment:
I usually add a sighting into the test setup too so the way a real encounter would exist is replicated. Probably not totally necessary.

Reply:
I think the sighting had no assets so didn't test the bit of code that outputs a list of assets in the json.